### ISSUE TITLE
[Issue]: Various common nouns tagged as proper noun.

### ISSUE THREAD

**Issue by MarketingPip** [spencermountain/compromise#1090]
It appears actor tags are causing tags to be tagged as `NNP (when using - Penn Tag)`. 

ie: "NNP" ```Proper Noun, Singular```

```js
function CompromiseTagger(word) {

    const doc = nlp(word);
    doc.compute('penn');
    const terms = doc.out('json')[0].terms[0];
    return terms.penn
}
console.log(CompromiseTagger("author"));
```

As far as I know these should be tagged as "Noun" / "NN"... 

I would assume the list needs cleaned up and a rule set needs implemented for somethings. Example "bishop" is in the list.

"Bishop" could be tagged as a proper noun if referring to a specific person's title, such as "Bishop John." or ```[#Actor] (#FirstName|#Person+)``` etc...

**Comment by MarketingPip** [comment]
@spencermountain - assuming you're using latest version of build...? And have you applied penn tags...? (I noticed some things were wrong - I'd have to reference some PDF's) but I think ordinal numbers are to be tagged as JJ. (I'll have to reference and confirm). 

ps; I pulled build from esm but I'll update you shortly to see if I didn't pull latest version or something? 🤷‍♂️

**Comment by MarketingPip** [comment]
@spencermountain - update. So Compromise (not using Penn tags), tags / chunk's it as a `Noun`. But when penn compute tags are applied turns into a ```NNP``` tag.

**Comment by MarketingPip** [comment]
No worries! I thought I was going crazy (trying to set up some demos of Compromise tagging some things in the HMM model I was showing you) until I started doing some digging hahahah! 

That said - hoping you'll be pumped up with HMM model (and maybe consider taking Compromise) that approach etc.. with some rules. Seeing some weirdly crazy good accuracy on tags (without rules) and with some basic rules + help of Compromise (and my half ass brain lol) - even crazier results. Tho < the rule set applied from Compromise. (after predicting tags) - should blow things out of the water.

**Comment by spencermountain** [comment]
fixed in `14.12.0` - thank you!
